Baby Monitor/Source Files/Custom Views/BaseSpinner.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  BaseSpinner.swift
+//  Baby Monitor
+//
+
+import UIKit
+
+protocol Spinner {
```

Comment:
Just a suggestion - it might be better to name it e.g. `ProgressIndicator` - Spinner might be used in different context on different platforms (https://developer.android.com/guide/topics/ui/controls/spinner), which might be misleading. On top of that, it doesn't limit the indicator to the shape of spinning thing only - not sure if that's relevant in this case though

Reply:
It's not a case here but I agree, thx 👍